tools/ew.py

```diff
@@ -263,7 +263,7 @@ def qemu_run(platform, machine, processor):
 		cmdline += ' -nographic'
 
 	if (not console and (not is_override('nographic')) and not is_override('noserial')):
-		cmdline += ' -serial stdio'
+		cmdline += ' -serial vc -monitor stdio'
```

Comment:
I actually preferred the old way of redirecting the serial console to stdio because then I could easily scroll and copy a paste it.

Reply:
Hmm, these changes should not be in this branch at all.
But this particular change I made actually to have both serial and monitor in separate text consoles. I think this is fairly recent I think, but QEMU does some very nice things with `vc` as of latest version.